firebase-app-distribution/src/main/java/com/google/firebase/app/distribution/TesterSignInManager.java

```diff
@@ -143,8 +136,16 @@ private void onActivityDestroyed(Activity activity) {
 
       signInTaskCompletionSource = new TaskCompletionSource<>();
 
-      alertDialog = getSignInAlertDialog(currentActivity);
-      alertDialog.show();
+      firebaseInstallationsApi
```

Comment:
Everything in signInTester() before this used to happen prior to the sign in dialog showing. Now it happens after the sign in dialog shows, right? Are there any implications of that? Seems fine to me but just wondering.

Reply:
We chatted this out, seems fine.